### DISCUSSION TITLE
How can I load custom .js library globally?

### DISCUSSION THREAD

**Discussion by mangelro** [quasarframework/quasar#14725] [General - Components / Directives / etc] [answered]
Is it possible **globally**  load in quasar v2.8.2 + vite v1.1.0 proyect a custom helper library ( .js) ?

Functions of this library would be available in all components without using the imports statement.

Mainly I would use to implements Array prototypes methods to help me in components logic.

Thanks in advance

**Comment by MilosPaunovic** [comment, marked as answer]
Use boot files.
https://quasar.dev/quasar-cli-webpack/boot-files#axios
